lxd/api_cluster.go

```diff
@@ -1615,6 +1623,99 @@ func clusterNodeDelete(d *Daemon, r *http.Request) response.Response {
 	return response.EmptySyncResponse
 }
 
+// swagger:operation POST /1.0/cluster/certificates cluster clustering_update_certs
+//
+// Update certificates for the cluster
+//
+// Replaces existing cluster certificates and reloads LXD on each cluster member
```

Comment:
Nit based on our current swagger styling but the long form description should end with a period.
The short form however doesn't so you shouldn't change that one :)